python/lsst/meas/modelfit/integrals.cc

```diff
@@ -21,21 +21,23 @@
  * see <http://www.lsstcorp.org/LegalNotices/>.
  */
 
+#include "pybind11/pybind11.h"
 
-%include "lsst/meas/modelfit/optimizer.i"
+#include "lsst/meas/modelfit/integrals.h"
 
-%{
-#include "lsst/meas/modelfit/GeneralPsfFitter.h"
-#include "lsst/meas/modelfit/DoubleShapeletPsfApprox.h"
-%}
+namespace py = pybind11;
+using namespace pybind11::literals;
 
-%include "lsst/meas/modelfit/GeneralPsfFitter.h"
-%include "lsst/meas/modelfit/DoubleShapeletPsfApprox.h"
+namespace lsst {
+namespace meas {
+namespace modelfit {
+namespace {
 
-%pythoncode %{
-import lsst.pex.config
+PYBIND11_PLUGIN(integrals) {
+    py::module mod("integrals");
+    mod.def("phid", &detail::phid);
+    mod.def("bvnu", &detail::bvnu);
```

Comment:
It's actually referenced as `lsst.meas.modelfit.detail.bvnu`, which works because `meas/modelfit/__init__.py` has:
```
from . import integrals as detail
```
I'm not sure that was wise, but it nicely makes the Python namespace match the C++ one while letting the wrapper source file match the header name.